maestro-cli/src/main/java/maestro/cli/device/Platform.kt

```diff
@@ -3,5 +3,11 @@ package maestro.cli.device
 enum class Platform(val description: String) {
     ANDROID("Android"),
     IOS("iOS"),
-    WEB("Web")
+    WEB("Web");
```

Comment:
Is this required? We don't support WEB anymore

Reply:
People still used and while we don't support it it's labeled as "Experimental". No reason to make any changes now.